specification/changeanalysis/resource-manager/Microsoft.ChangeAnalysis/preview/2020-04-01-preview/changeanalysis.json

```diff
@@ -0,0 +1,432 @@
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json",
+          "default"
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderOperationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceIdentity": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "None",
+            "SystemAssigned"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ManagedIdentityTypes",
+            "modelAsString": true
+          }
+        },
+        "principalId": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true
+        },
+        "tenantId": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AzureMonitorWorkspaceProperties": {
+      "type": "object",
+      "properties": {
+        "workspaceId": {
+          "type": "string"
+        },
+        "workspaceResourceId": {
+          "type": "string"
+        }
+      }
+    },
+    "NotificationsState": {
+      "enum": [
+        "None",
+        "Enabled",
+        "Disabled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "NotificationsState",
+        "modelAsString": true
+      }
+    },
+    "NotificationSettings": {
+      "type": "object",
+      "properties": {
+        "azureMonitorWorkspaceProperties": {
+          "$ref": "#/definitions/AzureMonitorWorkspaceProperties"
+        },
+        "activationState": {
+          "$ref": "#/definitions/NotificationsState"
+        }
+      }
+    },
+    "ConfigurationProfileResourceProperties": {
+      "type": "object",
+      "properties": {
+        "notifications": {
+          "$ref": "#/definitions/NotificationSettings"
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "readOnly": true
+        },
+        "lastModifiedTime": {
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdTime": {
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ConfigurationProfileResource": {
+      "description": "The configuration profile along with all resource properties.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
```

Comment:
There are routing implications for "top-level" proxy resources (i.e. no location based routing). I assume this is intentional?

Reply:
Yes this is intentional , we only need few properties from user configuration for notification changes. We discussed this with @Anthony Martin and he suggested that proxy should be sufficient for what we need.